dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLVectorStoreOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBNoSQL;
+
+/// <summary>
+/// Options when creating a <see cref="AzureCosmosDBNoSQLVectorStore"/>.
+/// </summary>
+public sealed class AzureCosmosDBNoSQLVectorStoreOptions
+{
+    /// <summary>
+    /// An optional factory to use for constructing <see cref="AzureCosmosDBNoSQLVectorStoreRecordCollection{TRecord}"/> instances, if custom options are required.
```

Comment:
I think it's correct, because with this factory we initialize the same `AzureCosmosDBNoSQLVectorStoreRecordCollection` class manually, which allows us to pass custom options. If factory is not provided, it will be initialized with default options. But let's double-check it with @westey-m.

Reply:
Taking into account, that factory can return any class that implements `IVectorStoreRecordCollection<TKey, TRecord>`, it can return custom record collection as well, so I've updated this documentation in all connectors as per your suggestion. Thanks!